tests/phpunit/FunctionalAbsTest.php

```diff
@@ -298,7 +316,7 @@ private function createTestProcess(?string $configId = null): Process
         );
     }
 
-    private function createConfigFile(string $blobPrefix): void
+    private function createConfigFile(string $blobPrefix, bool $restoreConfigs = true): void
```

Comment:
Chtěl jsem ti to napsat už u ty migrate appky :) Hrozně nemám rád tady ty defaulty v pomocných metodách 

Ono to jako vypadá neškodně (a v tomhle případě celkem je), ale po pár letech to dopadne takhle https://github.com/keboola/output-mapping/blob/293bc6b13b0319b2eb605ec14a4cf897d5f593aa/tests/AbstractTestCase.php#L126 a good luck finding co se v takovým testu děje.